cmd/crc/cmd/console.go

```diff
@@ -23,11 +23,13 @@ import (
 )
 
 var (
-	consoleURLMode bool
+	consoleURLMode          bool
```

Comment:
`consoleURLPrintMode` ?
vague in meaning

Reply:
The PR was merged already.

Reply:
I guess should just be `consolePrintURL`